mne/viz/_brain/_timeviewer.py

```diff
@@ -491,6 +505,24 @@ def set_slider_style(self, slider, show_label=True):
             if not show_label:
                 slider_rep.ShowSliderLabelOff()
 
+    def help(self):
+        shortcuts = {
+            'h': 'Display help window',
+            'y': 'Toggle interface',
+            't': 'Apply auto-scaling',
+            'u': 'Restore original clim',
+            'Space': 'Start/Pause playback'
+        }
+        text = [str(s) + " : \n" for s in shortcuts.keys()]
```

Comment:
Dict order is only guaranteed in Python 3.6+, we still support 3.5. You can probably just turn this into a tuple, then do
```
pairs = [
    ('h', 'Display help window'),
    ...
]
text1, text2 = zip(*pairs)
text1 = '\n'.join(text1)
text2 = '\n'.join(text2)
```